i18n/en/pop_desktop_widget.ftl

```diff
@@ -93,11 +93,11 @@ top-bar = Top Bar
 
 window-controls = Window Controls
 
-workspace-picker-position = Placement of the Workspace Picker
+workspace-picker-position = Position of the Workspace Picker
```

Comment:
This is the only sentence that uses "placement" instead of "position". It makes more sense to use the same word everywhere.